### PR TITLE
Code cleaning and small improvements

### PR DESCRIPTION
**Pull request by henrydays**

### CODE REVIEW

File: src/reporters.js [hunk 1]
```diff
@@ -202,26 +138,27 @@ function reportElastic(stats, elasticReporter, repository) {
 
             if (_.isObject(repoObj)) {
                 time = utilities.getDateForCommit(utilities.getRepoFolder(repoObj), repo.hash);
-                if (_.isObject(repository)) {
+                console.log(time);
```
Comment (lfac-pt) [inline src/reporters.js]: Remove this?

File: src/reporters.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ function repoToConsoleLine(repo) {
             if (metric.info.name === metricNames[i]) {
                 if (metric.result === null) {
                     logger.err("error on table");
-                } else if (metric.result.result === undefined) {
+                } else if (metric.result == undefined || metric.result.result === undefined) {
```
Comment (lfac-pt) [inline src/reporters.js]: Can we change this to `else if (!_.isObject(metric.result) || !_.isObject(metric.result.result))`?

File: src/reporters.js [hunk 3]
```diff
@@ -251,8 +188,9 @@ function reportElastic(stats, elasticReporter, repository) {
  * @param  {Object} stats
  */
 function report(stats, repo) {
-    createIndexes(stats);
 
+    console.log("repo nos reports");
```
Comment (lfac-pt) [inline src/reporters.js]: Remove this?

File: src/reporters.js (hunk 2 above)
Comment (henrydays) [reply src/reporters.js]: yea, sure

**Pull request merged**